packages/reporter/src/attempts/attempt-model.ts

```diff
@@ -25,7 +25,7 @@ export default class Attempt {
   @observable _state?: TestState | null = null
   @observable _invocationCount: number = 0
   @observable invocationDetails?: FileDetails
-  @observable hookCount: { [name in HookName]: number } = {
+  @observable hookCount: Record<HookName, number> = {
```

Comment:
why is this change needed?

Reply:
The linting was complaining about `name` not being used.
Prior to this PR no linting was done at build time.